packages/components/src/Main/Main.tsx

```diff
@@ -1,110 +1,24 @@
-import React, { ReactElement } from 'react';
+import React, { useEffect } from 'react';
 import classNames from 'classnames';
-import { connect } from 'react-redux';
-import { DarkContext } from '../Dark';
 import './main.scss';
-import type { ChromeAPI } from '@redhat-cloud-services/types';
 
-declare global {
-  interface Window {
-    insights: {
-      chrome: ChromeAPI;
-    };
-  }
-}
-
-const toKebab = (text: string) => text.replace(/([a-z0-9])([A-Z])/g, '$1-$2').toLowerCase();
-
-export interface InternalMainProps {
-  params?: { [key: string]: string };
-  path?: string;
-  className?: string;
-  children?: React.ReactNode;
-}
-
-type FcalculateLocation = () => { staticPart: string[]; dynamic: undefined | { [key: string]: any } };
+export type MainProps = React.DetailedHTMLProps<React.HTMLAttributes<HTMLElement>, HTMLElement>;
+export type InternalMainProps = MainProps;
 
 /**
- * This is a component that wraps the page
+ * @deprecated Do not use this component, use HTML section tag
  */
-export const InternalMain: React.FunctionComponent<InternalMainProps> = ({ path, params = {}, children, className, ...props }) => {
-  const calculateLocation: FcalculateLocation = () => {
-    if (window?.insights?.chrome?.$internal?.store) {
-      const chromeState = window.insights.chrome.$internal.store.getState();
-      if (path && chromeState) {
-        return path.split('/').reduce(
-          (acc, curr) => {
-            if (curr.indexOf(':') === 0) {
-              acc.dynamic = {
-                ...acc.dynamic,
-                [`data-${toKebab(curr.substr(1))}`]: params[curr.substr(1)],
-              };
-            } else {
-              acc.staticPart = [...acc.staticPart, ...(curr !== '' ? [curr] : [])];
-            }
-
-            return acc;
-          },
-          { staticPart: [chromeState.chrome.appId], dynamic: {} }
-        );
-      }
-    }
-
-    return {
-      staticPart: [],
-      dynamic: undefined,
-    };
-  };
-
-  const { dynamic, staticPart } = calculateLocation();
+export const Main: React.FC<MainProps> = ({ children, className, ...props }) => {
+  useEffect(() => {
+    console.error(`Using deprecated "Main" component. Do not use it. Either remove it from your JSX or replace it by "section" HTML element.`);
```

Comment:
It would make sense to have a eslint check for this rather than log output.

Maybe there could be a general "check for deprecated FEC stuff" and have a list/config in the shared eslint for those function/component names.

Reply:
I can do that as a follow. We can add it to the existing rule. But it's not a trivial fix. I will make a note to work on it.

Reply:
@Hyperkid123 That would be awesome! Thank you! I think we don't necessarily need a fix or even make it an error. It can just be a warning without a fix and only suggest to remove or replace it. The eslint errors are a bit more "in your face" since they'll pop up everywhere. :D